src/tm_mad/ceph/postmigrate

```diff
@@ -36,6 +36,21 @@ TEMPLATE_64=$6
 
 #--------------------------------------------------------------------------------
 
+# Get custom datastore values
+DRIVER_PATH=$(dirname "$0")
+XPATH="${DRIVER_PATH}/../../datastore/xpath.rb --stdin"
+
+unset XPATH_ELEMENTS i j
+
+while IFS= read -r -d '' element; do
+  XPATH_ELEMENTS[i++]="$element"
+done < <(onedatastore show -x "$DSID" | $XPATH \
+  /DATASTORE/TEMPLATE/SHARED)
+
+SHARED="${XPATH_ELEMENTS[j++]:-$SHARED}"
```

Comment:
Well I've learned it the hard way - I wanted to have single datastore TM_MAD for images and system, but only the images to be shared. So I've tried to edit the SHARED=no in system's record and it was (magically) reverted to SHARED=YES as it was set-up in the /etc/one/oned.conf . Ithink it should be override-able per datastore but I've no time so I made a workaround in our addon.